src/ode23s.jl

```diff
@@ -56,15 +56,15 @@ function show(io::IO, state :: RosenbrockState)
 end
 
 
-function start{O<:ExplicitODE,S<:ModifiedRosenbrockStepper}(s :: Solver{O,S})
+function start{O<:ExplicitODE,S<:ModifiedRosenbrockStepper}(s::Solver{O,S})
     t  = s.ode.t0
     dt = s.options.initstep
     y  = s.ode.y0
     dy = zero(y)
 
     J  = Array(eltype(y),length(y),length(y))
 
-    step  = Step(t,deepcopy(y),deepcopy(dy))
+    step  = Step(t,copy(y),copy(dy))
```

Comment:
It is type-stable on Julia 0.5.  Actually, I think it makes sense to tune this PR to 0.5 as it will be out by the time this is finished.  In particular the new functions will have a major impact on how ODE.jl performs. (In a pinch, we could even drop 0.4 support.)